### ISSUE TITLE
General Tracking Changes

### ISSUE THREAD

**Issue by shieldpoke** [TheGrandArchives/DS3CommunityPatch#27]
A common complaint is that the tracking of weapons is far too strong, I think the meta would greatly benefit if it was harder to just auto track an enemy down like a heat missile. But not all weapons need this change, maybe only curved swords and straight swords. Some classes need some tracking buffs as well. I think this would promote unlocked play a tiny bit more, making the game deeper overall.

**Comment by Ziggylata** [comment]
So are you one of those ds2 players who spins in the middle of combat constantly and bores me out of my mind to the point where I just open up cheat engine and kill myself?

**Comment by shieldpoke** [comment]
You mean I play unlocked? Yes, I do play unlocked. Super tracking being nerfed would be a great indirect buff to other weapons anyway.

**Comment by SpartanHexus** [comment]
I don't mind it being done. Which weapons you suggest a tracking increase?

**Comment by shieldpoke** [comment]
If pikes aren't going to be removed, definitely pikes. A tiny bit on halberds. A massive amount on great axes. Moderate tracking increase on estoc. But 99% of them are going to be tracking nerfs and not buffs. Straight swords and curved swords are going to get fucked. If it's possible, nerf unlocked play on spears a tiny tiny bit. Reapers need a tracking buff as well. IDK, its just a mess of ideas rn

**Comment by SpartanHexus** [comment]
Yeah pikes seriously needs a tracking buff. Straight swords and curved swords could use less tracking. But maybe let the tracking unchanged on the painting guardian sword for example, the dagger range curved sword doesn't need to be worse...
